examples/snippets/.env.example

```
@@ -1,10 +1,21 @@
-# Your Ethereum private key (without 0x prefix)
-PRIVATE_KEY=
-# URL of the RPC node to the target chain
+# Required: Author private key for testing
+AUTHOR_PRIVATE_KEY=
+
+# Optional: App private key for testing
+APP_PRIVATE_KEY
```

dotenv-linter (3.3.0)

[warning] [KeyWithoutValue] The APP_PRIVATE_KEY key should be with a value or have an equal sign

(KeyWithoutValue)